flask_simplelogin/__init__.py

```diff
@@ -99,7 +101,8 @@ def dispatch(fun, *args, **kwargs):
         elif is_logged_in():
             return SimpleLogin.get_message('access_denied'), 403
         else:
-            flash(SimpleLogin.get_message('login_required'), 'warning')
+            message = SimpleLogin.get_message('login_required')
+            message.flash()
```

Comment:
I'm still afraid this will not work with customized messages (and we have no test for that). If `SimpleLogin` is not instantiated, how are we reading the custom message?

Reply:
Oh. You're absolutely right. I assumed that `get_message()` was returning a `Message` object. I see that 102 is expecting a string, but line 105 is expecting the `Message` object. I'll work on that.
I'll probably change 102 to `Simplelogin.get_message('access_denied').text` and then make `get_message()` return an object (with `self.text` formatted to include `args` and `kwargs`.)

Reply:
That was not my point. The point is that in `SimpleLogin.get_message('login_required')` the class `SimpleLogin` wasn't instantiated with a custom `messages` dictionary.